### ISSUE TITLE
LaTeX error: destination with the same identifier (name{page.i}) has been already used, duplicate ignored

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1249]
I get warnings like this when compiling Sphinx documents:

```
pdfTeX warning (ext4): destination with the same identifier (name{page.i}) has been already used, duplicate ignored
```

PDFs have page numbering i, ii, i, ii, 1. The first page i is the title, while the second is the table of contents.

One way to avoid a duplication would be to change the starting page numbering in `sphinxmanual.cls` from `\pagenumbering{roman}` to `\pagenumbering{alph}`, and add `\pagenumbering{roman}&#37;` to the beginning of the definition of `\tableofcontents` Then the title page and its verso would have blank page numbers, and i and ii would be unique for the table of contents.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1249
- Originally reported by: [michaelhoffman](https://bitbucket.org/michaelhoffman)
- Originally created at: 2013-08-30T04:10:39.203

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-12 22:01:24+00:00_

Thanks for the suggestion!

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-12 22:01:37+00:00_

Closes #1249: Fix duplicate LaTeX page numbering for manual documents.

→ <<cset 5d23ef5a8c25d592672d556580c821f02c23feaf>>
